_posts/2023-16-15-Second-Issue-Newsletter

```diff
@@ -0,0 +1,292 @@
+
+Only dashboard (black X on a red background) errors cause non-compliance. Warnings (yellow triangle) or information (blue i) in the final “Summary” column are ok.
+
+An explanation of the dashboard report is available here ([link to dashboard video](https://youtu.be/m2khZcJVKU0?feature=shared&t=1319)).
+
+The “Maintained” column is dedicated to indicating recent updates to content in the ontology, and has been changed to avoid an error state.  A warning is shown for [inactivity](https://obofoundry.org/docs/OntologyStatus.html) beyond 2 years; an information icon is provided for 1-2 year [inactivity ](https://obofoundry.org/docs/OntologyStatus.html) Note that the [OBO Maintenance principle](https://obofoundry.org/principles/fp-016-maintenance.html) also asks that each ontology keep up to date with scientific consensus about its content, but this is not evaluated in the dashboard.
+
+Moving forward, we will provide some guidance on how to pass specific errors on the OBO Dashboard. In this issue, we will explain how to pass the “Open” principle. For details, see “Spotlight on OBO Principles”.
+
+
+---
+
+## Decisions Made and Important Updates
+
+
+
+* All terms in New Ontology Requests MUST follow the OBO identifier scheme (often they are accidentally written wrongly, e.g. using https instead of http). There MUST NOT be a term with the same meaning available in another OBO Foundry ontology, ie there must not be a term referring to a concept that already exists in another OBO Foundry ontology (whether or not the label is identical). There SHOULD NOT be another OBO Foundry ontology whose scope covers any of the new terms. See[ Standard Operating Procedures](https://obofoundry.org/docs/SOP.html#ROOM).
+* NCIT does NOT count when term overlap with other ontologies is determined. This is relevant because new ontologies are not allowed to create new terms for concepts that already exist in OBO. [See meeting of 27th June 2023](https://docs.google.com/document/d/1qhLFQL5IzTMUIBOtxJ5AqaEHcOCOQgNeXfvAb5O5P0A/edit).
+* [Ontologies which redirect to OLS are now redirecting to OLS4.](https://github.com/OBOFoundry/purl.obolibrary.org/pull/922)
+
+
+---
+
+## Ongoing Discussions
+
+Here, we list some of the more important discussions happening around the OBO-sphere:
+
+
+
+* The OBO community discusses the best approach to managing COB identifiers (IDs) and mappings. The discussion revolves around the question of whether COB should mint new identifiers in the COB namespace, or re-use identifiers from existing OBO ontologies. Both have advantages and disadvantages. [Join the discussion.](https://github.com/OBOFoundry/COB/issues/244)
+* This year, we have frequently stumbled across the question of whether an OBO ontology needs to be somehow related to the Biomedical or Biological domain or not. [Give us your opinion!](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1918)
+
+
+---
```

Comment:
section breaks are redundant of hierarchical titles

Reply:
What does this mean?

Reply:
we don't need `---` between sections, since we use headers like `##` and `###` which imply a hierarchy and sectioning